### ISSUE TITLE
HTTP server module

### ISSUE THREAD

**Issue by talentlessguy** [elsaland/elsa#51]
Once an HTTP server will be ready I will be able to make an [Elsa port](https://github.com/talentlessguy/tinyhttp/issues/89) of my web framework, so creating this issue to track the progress

**Comment by littledivy** [comment]
Thanks for opening an issue. We're currently working on implementing an http server #35

**Comment by talentlessguy** [comment]
@littledivy nice! with the issue it'll be a bit easier to track the progress ^_^

**Comment by talentlessguy** [comment]
Looks like it got implemented?

**Comment by littledivy** [comment]
Yes, it was added as `Elsa.serve`, but async interop with Quickjs is very poor atm so consider it to be unstable.
